src/utils/discord_utils.py

```diff
@@ -277,14 +281,18 @@ def get_url(content, accept_emojis=True, accept_templates=True):
         if len(results) != 0:
             emoji_id = results[0][2]
             is_animated = results[0][0] == "a"
-            url = "https://cdn.discordapp.com/emojis/{}.{}".format(
+            url = "https://media.discordapp.net/emojis/{}.{}".format(
                 emoji_id, "gif" if is_animated else "png"
             )
             return dict(url=url, type="emoji")
 
     return None
 
 
+def get_image_url(image: _EmbedMediaProxy) -> str:
```

Comment:
Hey thank you for your PR! I'm not sure I understand why using the proxy URL is better but I trust you on this.

I do have an issue with your implementation though, it's probably a version issue (I tested with python 3.10.7) but this type parameter line 2 causes an error for me: `NameError: name '_EmbedMediaProxy' is not defined`. Why not use `disnake.Attachment` for the type instead?

Reply:
Ok I didn't really run the code, just tried to type check it a bit and didn't realize that disnake was using `from __future__ import annotations` to make their code work.

The types that are passed in this function are `_EmbedMediaProxy`, `Attachment`, or even just `Embed`. `_EmbedMediaProxy` is probably the most accurate but then we're importing a kinda private member which doesn't feel right.

So for now I'm just removing the annotation completely.

Reply:
Yeah, I guess removing the annotation is a way to fix this, it works now!